
From: Hans Reiser <reiser@namesys.com>
Subject: Add reiser4 to various makefiles and Kconfig

This patch involves reiser4 into compiling (if it is configured)

Signed-off-by: Andrew Morton <akpm@osdl.org>
Acked-by: Chris Mason <mason@suse.com>
---

 fs/Kconfig  |    2 ++
 fs/Makefile |    1 +
 2 files changed, 3 insertions(+)

diff -puN fs/Kconfig~reiser4-include-reiser4 fs/Kconfig
--- 25/fs/Kconfig~reiser4-include-reiser4	2005-06-25 01:17:26.000000000 -0700
+++ 25-akpm/fs/Kconfig	2005-06-25 01:17:26.000000000 -0700
@@ -176,6 +176,8 @@ config FS_MBCACHE
 	default y if EXT2_FS=y || EXT3_FS=y
 	default m if EXT2_FS=m || EXT3_FS=m
 
+source "fs/reiser4/Kconfig"
+
 config REISERFS_FS
 	tristate "Reiserfs support"
 	help
diff -puN fs/Makefile~reiser4-include-reiser4 fs/Makefile
--- 25/fs/Makefile~reiser4-include-reiser4	2005-06-25 01:17:26.000000000 -0700
+++ 25-akpm/fs/Makefile	2005-06-25 01:17:26.000000000 -0700
@@ -52,6 +52,7 @@ obj-$(CONFIG_PROFILING)		+= dcookies.o
 # Do not add any filesystems before this line
 obj-$(CONFIG_FSCACHE)		+= fscache/
 obj-$(CONFIG_REISERFS_FS)	+= reiserfs/
+obj-$(CONFIG_REISER4_FS)	+= reiser4/
 obj-$(CONFIG_EXT3_FS)		+= ext3/ # Before ext2 so root fs can be ext3
 obj-$(CONFIG_JBD)		+= jbd/
 obj-$(CONFIG_EXT2_FS)		+= ext2/
_
